RNTester/js/AccessibilityExample.js

```diff
@@ -109,7 +109,7 @@ class AccessibilityExample extends React.Component {
           <TouchableOpacity
             onPress={() => Alert.alert('Button has been pressed!')}
             accessibilityRole="button"
-            accessibilityStates={['disabled']}
+            accessibilityStates={{enabled: false}}
```

Comment:
Let's not change from `disabled` to `enabled`. This should be `disabled: true`

Reply:
#24095  too.

Reply:
@necolas Not sure about that. I think iOS is the only platform that defines this state in the negative. Wouldn't it be better to align with the rest of the world on this? Especially if we're moving to a new API.

Reply:
@necolas Not sure about that. :) As far as I know, iOS is the only platform that defines this state in the negative (disabled instead of enabled). Shouldn't we take this opportunity to align with the majority of the platforms, especially given that we're proposing a braking API change anyway?

Reply:
Web uses `disabled` too. Let's leave this as `disabled`

Reply:
I agree with @necolas.

Reply:
Done.